Samples/blazor-add-in/outlook-blazor-add-in/README.md

```diff
@@ -0,0 +1,131 @@
+---
+page_type: sample
+urlFragment: outlook-blazor-add-in
+products:
+  - office-add-ins
+  - office
+languages:
+  - javascript
+  - C#
+extensions:
+  contentType: samples
+  technologies: Add-ins
+  createdDate: '08/25/2022 11:13:00 PM'
+description: 'Create a Blazor Webassembly Outlook add-in showcasing some samples.'
+---
+
+# Create a Blazor Webassembly Outlook add-in
+
+This sample shows how to build an Outlook add-in using .NET Blazor technologies. Blazor Webassembly allows you to build Office Add-ins using .NET, C#, and JavaScript to interact with the Office JavaScript API. The add-in uses JavaScript to work with the document and Office JS APIs, but you build the user interface and all other non-Office interactions in C# and .NET Core Blazor technologies.
+
+Working with Blazor Webassembly in the framework of Office, you can:
+- Build cross-platform Office Add-ins using Blazor, C#, and JavaScript Interop
+- Initialize the Office JavaScript API library in Blazor context
+- Interact with Outlook to manipulate paragraphs and content controls
+- Interact with document content through Office JavaScript APIs
```

Comment:
Probably remove this line since it is focused on Word or Excel.

Reply:
Done